gitrevise/todo.py

```diff
@@ -242,27 +242,63 @@ def edit_todos(
     return result
 
 
+def is_fixup(todo: Step) -> bool:
+    return todo.kind in (StepKind.FIXUP, StepKind.SQUASH)
```

Comment:
Nitpick: Is "squash" or "fixup" the generalization of the two? I would say "squash", since you can say

> I squashed my fixups.